.travis.yml

```diff
@@ -0,0 +1,22 @@
+---
+language: node_js
+node_js:
+  - "7"
+cache:
+  directories:
+  - frontend/node_modules/
```

Comment:
This probably the recommended way of doing it: https://docs.travis-ci.com/user/caching#npm-cache.

Reply:
I am not even sure if we should do caching. Does it really take that long? If it doesn't take too long, caching might just cause more headache than without it.

Reply:
It seems to be working fine, I was doing builds every few minutes so it saved a bunch of time, I can probably remove it now though.